js/data.js

```diff
@@ -135,7 +149,7 @@ var data = {
 		kurz: "BD Analytics",
 		lehrform: ["Vorlesung", "Übung"],
 		modul: ["Vertiefungsgebiete"],
-		semester: [ws15_16],
+		semester: [ws15_16, ws16_17],
```

Comment:
For all those cases, were you just added "ws16_17": Have you checked, that the Vertiefungsgebiete match the ones from the earlier semesters?